.gitignore

```diff
@@ -1,3 +1,7 @@
 /node_modules/
 /npm-debug.log
 tmp/
+
+# files generated at build time
+bin/uglifyjs
```

Comment:
That's the CLI people use when installing `uglify-js` globally, so you don't want to ignore/rename this.

Reply:
This is just to prevent it from being committed to git, a copy of this file is included in the package and available when users install. Or did I miss something?

Reply:
So I think we should leave the command-line script as is, due to the same reason as https://github.com/mishoo/UglifyJS/pull/5765#discussion_r1051881389

Reply:
you want to overwrite `bin/uglifyjs` with the built `bin/uglifyjs` when building?

Reply:
I don't see any reason to process `bin/uglifyjs` in the first place &minus; just include it into the `npm` package untouched.

Reply:
I built it so that `require("../tools/tty")` is included. What do you propose for that instead?

Reply:
By "included" do you mean `tools/tty.js` is embedded into the newly generated `bin/uglifyjs`?

If so, I'm pretty certain that's not the case here &minus; please double-check the generated file.

Reply:
Oh, your right. So we'd need to include that file, too